desktop/linux/index.md

```diff
@@ -1,104 +1,172 @@
+You can also move the disk image to a different location. If you attempt to move a disk image to a location that already has one, you get a prompt asking if you want to use the existing image or replace it.
 
-Alternatively, open a terminal and run:
+#### File sharing
+
+Use File sharing to allow local directories on the Linux host to be shared with Linux containers.
+This is especially useful for
+editing source code in an IDE on the host while running and testing the code in a container.
+By default the `/home/<user>` directory is shared. If your project is outside this directory then it must be added
+to the list. Otherwise you may get `Mounts denied` or `cannot start service` errors at runtime.
+
+File share settings are:
+
+- **Add a Directory**: Click `+` and navigate to the directory you want to add.
+
+- **Remove a Directory**: Click `-` next to the directory you want to remove
+
+- **Apply & Restart** makes the directory available to containers using Docker's
+  bind mount (`-v`) feature.
+
+> Tips on shared folders, permissions, and volume mounts
+>
+ * Share only the directories that you need with the container. File sharing
+ introduces overhead as any changes to the files on the host need to be notified
+ to the Linux VM. Sharing too many files can lead to high CPU load and slow
+ filesystem performance.
+>
+ * Shared folders are designed to allow application code to be edited 
+ on the host while being executed in containers. For non-code items
+ such as cache directories or databases, the performance will be much 
+ better if they are stored in the Linux VM, using a [data volume](../../storage/volumes.md)
+ (named volume) or [data container](../../storage/volumes.md).
+
+#### Proxies
+
+Docker Desktop detects HTTP/HTTPS Proxy Settings and automatically
+propagates these to Docker. For example, if you set your
+proxy settings to `http://proxy.example.com`, Docker uses this proxy when
+pulling containers.
+
+If you want to configure proxies manually, turn on the **Manual proxy configuration** setting.
+
+Your proxy settings, however, will not be propagated into the containers you start.
+If you wish to set the proxy settings for your containers, you need to define
+environment variables for them, just like you would do on Linux, for example:
 
 ```console
-$ systemctl --user stop docker-desktop
+$ docker run -e HTTP_PROXY=http://proxy.example.com:3128 alpine env
+
+PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+HOSTNAME=b7edf988b2b5
+TERM=xterm
+HOME=/root
+HTTP_PROXY=http://proxy.example.com:3128
 ```
 
+For more information on setting environment variables for running containers,
+see [Set environment variables](/engine/reference/commandline/run/#set-environment-variables--e---env---env-file).
+
+#### Network
+
+You can configure Docker Desktop networking to work on a virtual private network (VPN). Specify a network address translation (NAT) prefix and subnet mask to enable Internet connectivity.
```

Comment:
```suggestion
Docker Desktop uses a private IPv4 network for internal services such as a DNS server and an HTTP proxy. In case the choice of subnet clashes with something in your environment, specify a custom subnet using the **Network** setting.
```